pkg/client/devops/jclient/jenkins.go

```diff
@@ -0,0 +1,49 @@
+package jclient
+
+import (
+	"io/ioutil"
+
+	client "github.com/jenkins-zh/jenkins-client/pkg/core"
+	"gopkg.in/yaml.v2"
+)
+
+type JenkinsClient struct {
+	
+}
+type Config struct {
+	Jenkins JenkinsConfig `yaml:"devops,omitempty"`
+}
+type JenkinsConfig struct {
+	URL      string `yaml:"host"`
+	UserName string `yaml:"username"`
+	Password string `yaml:"password"`
+	Token    string `yaml:"token,omitempty"`
+}
+
+func GetJenkinsCore() (core client.JenkinsCore, e error) {
```

Comment:
IMO, we can initialize `client.JenkinsCore` in `JenkinsClient` instead of reading configuration file, creating JenkinsCore and getting crumb at every invoke.